components/Form/Form.js

```diff
@@ -0,0 +1,188 @@
+            <span>Me</span>{" "}
+          </label>
+          <Checkbox
+            type="radio"
+            name="for"
+            defaultValue="Someone else"
+            onChange={handleForChange}
+          />{" "}
+          <span>Someone else</span>
+          <br />
+          {forSomeoneElse && (
+            <SomeoneElse>
+              <Label htmlFor="name">
+                {" "}
+                Name :
+                <Input type="text" name="name" />
+              </Label>
+              <Label htmlFor="age">
+                {" "}
+                Age :
+                <Input type="number" name="age" />
+              </Label>
+              <Label htmlFor="gender">
+                {" "}
+                Gender :
+                <Select name="gender">
+                  <option value="Male">Male</option>
+                  <option value="Female">Female</option>
+                  <option value="Other">Other</option>
+                </Select>
+              </Label>
+            </SomeoneElse>
+          )}
+          <Label htmlFor="importance">
+            {" "}
+            Importance :
+            <Select name="importance">
+              <option defaultValue="High">High</option>
+              <option defaultValue="Medium">Medium</option>
+              <option defaultValue="Low">Low</option>
+            </Select>
+          </Label>
+          <Label htmlFor="medication_name">
+            {" "}
+            Medication Name :
+            <Input type="text" name="medication_name" />
+          </Label>
+          <Label htmlFor="activ_ingredients">
+            {" "}
+            Activ Ingredients :
+            <Input type="text" name="activ_ingredients" />
+          </Label>
+          <Label htmlFor="dosage">
+            {" "}
+            Dosage :
+            <Input
+              type="number"
+              name="dosage"
+              id="dosage"
+              min="0"
+              step="0.01"
+            />
+          </Label>
+          <Label htmlFor="reason">
+            {" "}
+            Reason for taking:
+            <Input type="text" name="reason" />
+          </Label>
+          <Label htmlFor="form">
+            {" "}
+            Form :
+            <Select name="form">
+              <option defaultValue="Tablet">Tablet</option>
+              <option defaultValue="Capsule">Capsule</option>
+              <option defaultValue="Syrup">Syrup</option>
+              <option defaultValue="Injection">Injection</option>
+              <option defaultValue="Powder">Powder</option>
+              <option defaultValue="Ointment">Ointment</option>
+              <option defaultValue="Syringe">Syringe</option>
+              <option defaultValue="Cream">Cream</option>
+              <option defaultValue="Drops">Drops</option>
+              <option defaultValue="Spray">Spray</option>
+              <option defaultValue="Other">Other</option>
+            </Select>
+          </Label>
```

Comment:
OK! That's a valid point.
And thank you for mentioning the use of value instead of defaultValue when working with options. Using value instead of defaultValue would make the code more intuitive and easier to read.